### DISCUSSION TITLE
Installation on Mac is not going anywhere...

### DISCUSSION THREAD

**Discussion by stariontech** [elementary/installer#694] [General]
![UAuUDrtsTcWt%BPX8ma4zQ](https://user-images.githubusercontent.com/12043819/220266459-66d17578-3cf5-46cf-9d44-7d82a7a57248.jpg)
I've seen several people mentioning the same issue, create an installer USB or DVD, boot from it, screen goes black and basically hangs.  No logo.  I did get some text that I don't understand before the screen goes black.

I haven't seen any mention of how this might be fixed or how to work around it other than to create a DVD instead of a USB stick.

Would really like to try out Elementary, but can't even get past the opening gate...

Any insight on what to do next?

**Comment by ryonakano** [comment]
I created a bootable USB from the latest 7.0 iso (`elementaryos-7.0-stable.20230129rc.iso`) using the dd command and tried it on my MacBook Air 11-inch, 2010. I saw the above error messages for a while but soon after that the boot splash screen showed up and the Installer launched successfully:

![IMG_0658](https://user-images.githubusercontent.com/26003928/220628380-ee3ac19b-433a-4806-b49b-185d8222134d.jpg)

Could you give us more info like the model of your Mac, how you created your USB installer, etc.?

**Comment by stariontech** [comment]
It is an iMac 21.5-inch, Mid 2010 3.6Ghz Intel Core i5, 16GB RAM, 500GB HD, ATI Radeon HD 4670 256MB
Installer was created from downloaded ISO using latest Etcher for OSX.

**Reply by ryonakano** [reply]
There are some similar issues reported by who tried Ubuntu on the same model of iMac, like this:

https://www.reddit.com/r/Ubuntu/comments/112rilm/black_screen_after_ubuntu_or_any_other_linux

So this would be a compatibility issue between the dGPU (ATI Radeon HD 4670) and Linux, and would not be related to the Installer of elementary OS or elementary OS itself.

As the author of above post tried there is a boot option `nomodeset` in Linux but I'm not sure if that works since my Mac model is different from yours…

**Comment by stariontech** [comment]
So basically I can't use elementary.  Or anything else except OSX.  That's not encouraging.
